connector/kinesis-asl/pom.xml

```diff
@@ -76,6 +76,12 @@
       <artifactId>jackson-dataformat-cbor</artifactId>
       <version>${fasterxml.jackson.version}</version>
     </dependency>
+    <dependency>
+      <groupId>javax.xml.bind</groupId>
+      <artifactId>jaxb-api</artifactId>
+      <version>${jaxb-api.version}</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
If we need this for testing, this looks like an independent issue. Could you spin off from this GitHub Action PR?

Reply:
This test dependency is only needed for those tests that would incur billing cost, which are not executed  by default. I removed it from this PR.